verification/trustpolicy/trustpolicy.go

```diff
@@ -0,0 +1,448 @@
+package trustpolicy
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"github.com/notaryproject/notation-go/internal/common"
+	"github.com/notaryproject/notation-go/verification/truststore"
+)
+
+// ValidationType is an enum for signature verification types such as Integrity,
+// Authenticity, etc.
+type ValidationType string
+
+// ValidationAction is an enum for signature verification actions such as
+// Enforced, Logged, Skipped.
+type ValidationAction string
+
+// VerificationLevel encapsulates the signature verification preset and its
+// actions for each verification type
+type VerificationLevel struct {
+	Name        string
+	Enforcement map[ValidationType]ValidationAction
+}
+
+const (
+	TypeIntegrity          ValidationType = "integrity"
+	TypeAuthenticity       ValidationType = "authenticity"
+	TypeAuthenticTimestamp ValidationType = "authenticTimestamp"
+	TypeExpiry             ValidationType = "expiry"
+	TypeRevocation         ValidationType = "revocation"
+
+	ActionEnforce ValidationAction = "enforce"
+	ActionLog     ValidationAction = "log"
+	ActionSkip    ValidationAction = "skip"
+)
```

Comment:
One `const` block for one type.
```suggestion
const (
	TypeIntegrity          ValidationType = "integrity"
	TypeAuthenticity       ValidationType = "authenticity"
	TypeAuthenticTimestamp ValidationType = "authenticTimestamp"
	TypeExpiry             ValidationType = "expiry"
	TypeRevocation         ValidationType = "revocation"
)

const (
	ActionEnforce ValidationAction = "enforce"
	ActionLog     ValidationAction = "log"
	ActionSkip    ValidationAction = "skip"
)
```